features/admin_creates_auction.feature

```diff
@@ -18,6 +18,7 @@ Feature: Admin creates auctions in the admins panel
     Then I should see the start time I set for the auction
     And I should see the end time I set for the auction
 
+  @freeze_time
```

Comment:
it's cool that this fixes it -- but do we know *why*? If not, I am worried that we are missing an underlying bug or something that this exposes...